mlflow/keras.py

```diff
@@ -734,25 +734,31 @@ def _get_early_stop_callback_attrs(callback):
             return None
 
     def _log_early_stop_callback_metrics(callback, history, metrics_logger):
-        if callback:
-            callback_attrs = _get_early_stop_callback_attrs(callback)
-            if callback_attrs is None:
-                return
-            stopped_epoch, restore_best_weights, patience = callback_attrs
-            metrics_logger.record_metrics({"stopped_epoch": stopped_epoch})
```

Comment:
The current implementation always logs `stopped_epoch` even when training didn't early stop, which might be confusing.